### PR TITLE
Omit messages for its own expired key

### PR DESCRIPTION
**Pull request by nikksan**

### CODE REVIEW

**Review by veseo — APPROVED** [review]

File: src/RedisIPCMessenger.ts [hunk 1]
```diff
@@ -110,6 +110,10 @@ export default class RedisIPCMessenger implements IPCMessenger {
         throw new Error(`Failed to map ${key} to a callback`);
       }
 
+      if (instance === this.instance) {
```
Comment (zupper) [inline src/RedisIPCMessenger.ts]: Why don't we move this closer to the  top of the block, to be able to short-circuit?

Comment (zupper) [inline src/RedisIPCMessenger.ts]: Also, aren't we comparing a `string` to an `Instance` here?

Comment (nikksan) [reply src/RedisIPCMessenger.ts]: both checks can abort the handling, now that the destructuring is replaced by `parts`  array, it makes sense to process the inidividual parts in the order they appear in the array

Comment (nikksan) [reply src/RedisIPCMessenger.ts]: Correct, fixed

**Pull request merged**